ASB November 2025 Security Patches integration

alavalas1 opened the pull request:
ASB November 2025 Security Patches integration

Integrating Google Android Security Bulletin Patches

Test done: STS r44 TCs Passed except 2 tests (Jira ID:133685)
1) android.security.sts.KernelLtsTest#testRequiredKernelLts_WARN
2) CVE_2025_26450 - android.security.cts.CVE_2025_26450
testPocCVE_2025_26450 (Jira ID : 134479)

Tracked-On: OAM-134369
Signed-off-by: alavala srinivasa reddy <alavala.srinivasa.reddy@intel.com>

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

alavalas1 committed:
ASB November 2025 Security Patches integration

Integrating Google Android Security Bulletin Patches

Test done: STS r44 TCs Passed except 2 tests (Jira ID:133685)
1) android.security.sts.KernelLtsTest#testRequiredKernelLts_WARN
2) CVE_2025_26450 - android.security.cts.CVE_2025_26450
testPocCVE_2025_26450 (Jira ID : 134479)

Tracked-On: OAM-134369
Signed-off-by: alavala srinivasa reddy <alavala.srinivasa.reddy@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
FAILURE: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

tanujtekriwal reviewed (approved):
LGTM

shyjumon-n reviewed (approved):
LGTM

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /android-ci-local/celadon_a15_master-merge/29
